+jrclust/+sort/@TemplateClustering/autoMerge.m

```diff
@@ -1,4 +1,26 @@
-function autoMerge(obj)
+function success = autoMerge(obj)
```

Comment:
This looks (almost) identical to the `autoMerge` in `DensityPeakClustering`. We can move them both to `Clustering` instead, so changes later don't have to be duplicated.